site/public/css/polls.css

```diff
@@ -116,6 +116,25 @@ form {
     margin-left: 0.5em;
 }
 
+#timer-inputs {
+    display: inline-block;
+}
+
+.poll-timer {
+    display: inline-block;
+    text-align: center;
+    margin-right: 0.5em;
```

Comment:
I would remove margin-right to align the text.

Reply:
Fixed, looks much better